test/parallel/test-fs-write.js

```diff
@@ -47,26 +45,35 @@ fs.open(fn, 'w', 0o644, common.mustCall(function(err, fd) {
     console.log('found: "%s"', found);
     fs.unlinkSync(fn);
     assert.strictEqual(expected, found);
-  }));
+  });
+
+  fs.write(fd, '', 0, 'utf8', function(err, written) {
```

Comment:
There probably needs to be a `common.mustCall()` for this callback as well.

Reply:
Like this:

```
const written = common.mustCall(function(err, written) {
    assert.strictEqual(0, written);
  });

  fs.write(fd, '', 0, 'utf8', written);
```

?

Reply:
Either inline or not, doesn't really matter.